src/lmrtfy.js

```diff
@@ -1,4 +1,15 @@
 class LMRTFY {
+    static async init() {
+      game.settings.register('lmrtfy', 'enableParchmentTheme', {
+        name: game.i18n.localize('LMRTFY.EnableParchmentTheme'),
+        scope: 'client',
+        config: true,
+        type: Boolean,
+        default: true,
+        onChange: (value) => window.location.reload()
```

Comment:
I'd prefer not to have to trigger a reload when this is saved, but as we need to style the wrapping element, this couldn't be done from the template scope - the CSS class needs to be applied at the top level, via the `defaultOptions` static, AFAICT there's no other hook for setting these classes at run-time (this is my first look at FVTT modules).

Reply:
Agreed.
I think if you close the window and reopen it, it would call the defaultOptions again. The fact that it's static, just means it can be called on the class without an object being created first, but it gets called every time the class is created.

I think setting it at runtime should be a simple `onChange: (value) => $(".lmrtfy,.lmrtfy-roller").toggleClass(".lmrtfy-parchment", value)`

Humm.. actually, the requestor is a singleton, so the class is created once, its opens won't change. So yeah, for now, let that be a reload, I'm ok with it.

Reply:
I have a change that will apply this dynamically at run-time by overriding the private `_renderOuter()` method and appending the class to the rendered content, which allows the settings change without reload, if that's an approach you'd be interested in taking.

Reply:
Now it's too complex for what it should be. A simpler solution would be `LMRTFY.requestor.options.classes.push/remove` (if requestor is defined)